tests/test_api_paper.py

```diff
@@ -124,10 +117,10 @@ def test_order_option_limit(self):
         )
         self.assertEqual(order["type"], "OPTION")
         self.assertEqual(order["id"], 0)
-        self.assertEqual(order["symbol"], "A     211106P50001000")
+        self.assertEqual(order["symbol"], "A     211114P50001000")
 
         status = dummy.fetch_option_order_status(order["id"])
-        self.assertEqual(status["symbol"], "A     211106P50001000")
+        self.assertEqual(status["symbol"], "A     211114P50001000")
```

Comment:
on my machine the `06` is actually `13` and on github it is `14`. Is this a numpy version issue? @tfukaza

Reply:
hm might be, I guess in that case which ever version github runs should be used.